src/Compilers/CSharp/Test/Semantic/Semantics/StructsTests.cs

```diff
@@ -624,7 +624,7 @@ public struct X1
                 Diagnostic(ErrorCode.ERR_NonPublicParameterlessStructConstructor, "X1").WithLocation(11, 5));
         }
 
-        [Fact]
+        [Fact(Skip = "PROTOTYPE(semi-auto-props): ERR_FeatureNotAvailableInVersion9 is no longer produced. Investigate why.")]
```

Comment:
>Skip = "PROTOTYPE(semi-auto-props): ERR_FeatureNotAvailableInVersion9 is no longer produced. Investigate why." [](http://example.com/codeflow?start=14&length=110)

Please do not skip legacy tests. The only reason to skip a legacy test is a crash, but then we should really understand why we crash.

Reply:
Un-skipped. They worked properly with the latest changes.